data/map.txt

```diff
@@ -30091,21 +30132,41 @@ planet Winter
 		fleet "Large Militia" 5
 
 planet "Wormhole Alpha"
-	description ""
 	spaceport ""
 	government "Wormhole Alpha"
+	wormhole "Wormhole Alpha"
+
+wormhole "Wormhole Alpha"
+	mappable
+	link "Ultima Thule" Waypoint
+	link Waypoint "Ultima Thule"
 
 planet "Wormhole Barren Alpha"
 	attributes "requires: quantum keystone"
-	description ``
+	wormhole "Wormhole Barren Alpha"
+
+wormhole "Wormhole Barren Alpha"
+	mappable
+	link Egeria Levana
+	link Levana Egeria
 
 planet "Wormhole Barren Beta"
 	attributes "requires: quantum keystone"
-	description ``
+	wormhole "Wormhole Barren Beta"
+
+wormhole "Wormhole Barren Beta"
+	mappable
+	link Caeculus Prosa
+	link Prosa Caeculus
 
 planet "Wormhole Link"
 	attributes "requires: quantum keystone"
-	description ``
+	wormhole "Wormhole Link"
+
+wormhole "Wormhole Link"
```

Comment:
We should consider demonstrating that the planet and wormhole can now be named separately. This one is a good candidate IMO.

Reply:
I honestly have no idea what I should name it. Do  you have any recommendations?